evennia/web/webclient/static/webclient/js/webclient_gui.js

```diff
@@ -239,16 +241,74 @@ function onDefault(cmdname, args, kwargs) {
 // in Firefox, there it's a standard error.
 function onBeforeUnload() {
     return "You are about to leave the game. Please confirm.";
-}
+}
+
+var unread = 0;

```

Comment:
All these functions should have a comment block explaining wthat they are there for (c.f. other callbacks in this section).